app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Ukrainian teachers and trainees coming to the UK"
+subcategory: Train to teach in England as a non-UK citizen
```

Comment:
@laurahermionetennant92 as Ukraine covers both trainees and teachers I think it's a little confusing having it in the train to teach section and it might make more sense to have its own separate blue subhead at the bottom of the page. Maybe something like If you're from Ukraine?